From d7eccbbae84b2ee7dbb756e60287c4b47071444e Mon Sep 17 00:00:00 2001
From: Jiri Kosina <jkosina@suse.cz>
Date: Wed, 6 Oct 2010 11:34:48 +0200
Subject: Revert "drivers/usb: Remove unnecessary return's from void
 functions" for musb gadget
Patch-mainline: v2.6.37
Git-commit: d7eccbbae84b2ee7dbb756e60287c4b47071444e

This is a partial revert of 7f26b3a7533b ("drivers/usb: Remove unnecessary
return's from void functions") as this hunk will vanish in USB tree completely.

Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
Signed-off-by: Jiri Kosina <jkosina@suse.cz>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/musb/musb_gadget.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/musb/musb_gadget.c
+++ b/drivers/usb/musb/musb_gadget.c
@@ -850,6 +850,7 @@ void musb_g_rx(struct musb *musb, u8 epn
 		DBG(3, "packet waiting for %s%s request\n",
 				musb_ep->desc ? "" : "inactive ",
 				musb_ep->end_point.name);
+	return;
 }
 
 /* ------------------------------------------------------------ */
